openapi-specs/schema.json

```diff
@@ -27,6 +27,22 @@
       "type": "object",
       "description": "permissions attached to a role.",
       "properties": {
+        "backup": {
+          "type": "object",
+          "description": "resources applicable for backup actions",
+          "properties": {
+            "backend": {
```

Comment:
**suggestion (if-minor):** i think woe could do enum here for allowed backends , wdyt ?

Reply:
Hmm, `backend` isn't an enum in other areas of `schema.json` so perhaps it's better to maintain consistency here. WDYT?